src/components/Reown/SendTransactionModal.js

```diff
@@ -0,0 +1,439 @@
+              <View style={styles.valueContainer}>
+                <Text style={styles.monospace}>
+                  {truncateTxId(input?.txId)} ({input?.index})
+                </Text>
+                {input?.txId && (
+                  <TouchableOpacity onPress={() => copyToClipboard(input.txId)}>
+                    <Text style={styles.copyButton}>{t`Copy ID`}</Text>
+                  </TouchableOpacity>
+                )}
+              </View>
+              <Text style={styles.labelText}>{t`Endereço`}</Text>
+              <TouchableOpacity
+                style={styles.addressContainer}
+                onPress={() => copyToClipboard(input?.address)}
+              >
+                <Text style={styles.monospace} numberOfLines={1}>
+                  {input?.address}
+                </Text>
+              </TouchableOpacity>
+            </View>
+          );
+        })}
+      </View>
+    );
+  };
+
+  const renderOutputs = () => {
+    if (!data?.data?.outputs || data.data.outputs.length === 0) {
+      return null;
+    }
+
+    return (
+      <View>
+        <Text style={styles.sectionTitle}>{t`Outputs`}</Text>
+        {data.data.outputs.map((output, index) => {
+          const outputKey = `output-${output.address || ''}-${output.token || ''}-${Math.random().toString(36).substr(2, 9)}`;
+          return (
+            <View
+              key={outputKey}
+              style={styles.itemContainer}
+            >
+              <View style={styles.itemHeader}>
+                <Text style={styles.itemTitle}>{t`Output ${index + 1}`}</Text>
+                <Text style={styles.valueText}>
+                  {formatValue(output?.value)} {getTokenSymbol(output?.token)}
+                </Text>
+              </View>
+              <Text style={styles.labelText}>{t`Endereço`}</Text>
+              <TouchableOpacity
+                style={styles.addressContainer}
+                onPress={() => copyToClipboard(output?.address)}
+              >
+                <Text style={styles.monospace} numberOfLines={1}>
+                  {output?.address}
+                </Text>
+              </TouchableOpacity>
+              {output?.data && (
+                <View>
+                  <Text style={styles.labelText}>{t`Data field`}</Text>
+                  <View style={styles.valueContainer}>
+                    <Text style={styles.monospace}>
+                      {output.data.join(',')}
+                    </Text>
+                  </View>
+                </View>
+              )}
+            </View>
+          );
+        })}
+      </View>
+    );
+  };
+
+  const renderChangeAddress = () => {
+    if (!data?.data?.changeAddress) {
+      return null;
+    }
+
+    return (
+      <View>
+        <Text style={styles.sectionTitle}>{t`Change Address`}</Text>
+        <View style={styles.itemContainer}>
+          <Text style={styles.labelText}>{t`Endereço`}</Text>
```

Comment:
Do you need this label even with the title above?

Reply:
Refactored, thanks